js/screens/Profile/Profile.js

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
-import { View, Button } from 'react-native';
+import { View, Text, Button } from 'react-native';
 import PropTypes from 'prop-types';
-import ProfileHeader from '../../components/ProfileHeader/ProfileHeader';
+import ProfileHeaderContainer from '../../components/ProfileHeader/ProfileHeaderContainer';
 import { styles } from './styles';
 
-const Profile = ({ navigation, signOut }) => {
+const Profile = ({ navigation, signOut, userData }) => {
   return (
     <View style={styles.container}>
-      <View style={styles.profileTab} />
-      <ProfileHeader />
-      <Button
-        title="My Applications"
-        onPress={() =>
-          navigation.navigate('Application')
-        }
-        style={styles.profileText}
-      />
-      <Button onPress={signOut} title="sign me out" />
+      <View style={styles.profileTab}>
+        <Text style={styles.profileText}>Profile</Text>
+        <Button
+          title="My Applications"
+          onPress={() => navigation.navigate('Application')}
+          style={styles.profileText}
+        />
+      </View>
+      <ProfileHeaderContainer userData={{ userData }} />;
+      <View>
+        <Button onPress={signOut} title="sign me out" />
+      </View>
     </View>
   );
 };
 
 Profile.propTypes = {
-  navigation: PropTypes.object,
-  signOut: PropTypes.func
+  navigation: PropTypes.object
+  // signOut: PropTypes.func
```

Comment:
Proptype is commented out for prop that still exists in the component. Also missing proptype validation for userData, and whether props are required or default props.